Data/DynamicInputPack.cs

```diff
@@ -71,6 +71,24 @@ public bool PreserveAspectRatio
                 OnPropertyChanged(nameof(PreserveAspectRatio));
             }
         }
+
+        private string _game_id = "";
+        public string GameID
+        {
+            get
+            {
+                return _game_id;
+            }
+            set
+            {
+                if (value.Length <= 6)
+                {
+                    _game_id = value.ToUpper();
```

Comment:
My suggestion would be to do this to-upper logic in `WriteGameID`.  That way if we ever need to read back from the value in the text box, the user isn't surprised when their string becomes caps :)

Reply:
I found it more intuitive for the user, the text field is equipped with on Property Changed, so that the changes are displayed directly when typing. If you type an "a", an "A" appears, it also accepts only 6 characters.

Validation rule is a good idea however I am unsure if it is only letters and numbers?